packages/types/src/countryIso.ts

```diff
@@ -0,0 +1,234 @@
+export type CountryIso =
```

Comment:
I'm not sure about the duplicated type here. Can we use a simple `string` type?
The correct approach would be to extract countryData.ts into clerk/shared and reuse that, but this looks unnecessary right now!

Reply:
@nikosdouvlis I removed the redundant type and used a type of string instead